<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EmailVerifyView</class>
 <widget class="QWidget" name="EmailVerifyView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="palette">
   <palette>
    <active/>
    <inactive/>
    <disabled/>
   </palette>
  </property>
  <property name="windowTitle">
   <string>OctoVault VPN</string>
  </property>
  <widget class="QLabel" name="lblBackgroundImg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>500</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>800</width>
     <height>500</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/img_background.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblTitle">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>125</y>
     <width>200</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FFFFFF;
}</string>
   </property>
   <property name="text">
    <string>Email verification</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblSubTitle1">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>300</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>we'll send a 4-digit verification code to your email</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancel">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>420</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 15px;
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnVerify">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>340</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:#FACA7F;
border-radius: 15px;
color:#000000;
}</string>
   </property>
   <property name="text">
    <string>Verify</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblSpinner">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>300</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lblLogo">
   <property name="geometry">
    <rect>
     <x>385</x>
     <y>50</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>226</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="leOneHolder" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
      </property>
      <widget class="QLineEdit" name="leOne">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="leTwoHolder" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
      </property>
      <widget class="QLineEdit" name="leTwo">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="leThreeHolder" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
      </property>
      <widget class="QLineEdit" name="leThree">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="leFourHolder" native="true">
      <property name="minimumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>40</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">QWidget{
background:transparent;
border: 1px solid #36353a;
border-radius: 10px;
}

QWidget:hover{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 10px;
}</string>
      </property>
      <widget class="QLineEdit" name="leFour">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>40</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <bold>true</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QLineEdit{
background:transparent;
padding-left:0px;
color: #36353a;
border-color: transparent;
}

QLineEdit:hover{
background:transparent;
padding-left:0px;
color: #FACA7F;
border-color: transparent;
}</string>
       </property>
       <property name="maxLength">
        <number>1</number>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lblLogoTwo">
   <property name="geometry">
    <rect>
     <x>335</x>
     <y>50</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../assets/assets.qrc">:/img/images/logo_text.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnChangeLogo">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnResend">
   <property name="geometry">
    <rect>
     <x>275</x>
     <y>380</y>
     <width>250</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
background:transparent;
border: 1px solid #FACA7F;
border-radius: 15px;
color:#FACA7F;
}</string>
   </property>
   <property name="text">
    <string>Resend</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../assets/assets.qrc"/>
 </resources>
 <connections/>
</ui>
